.travis.yml

```
@@ -20,8 +23,12 @@ jobs:
       python: 3.9
       script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
 
-    - name: "Run tests on Python 3.10 (beta)"
-      python: 3.10-dev
+    - name: "Run tests on Python 3.10.2"
```

Oh, forgot the action title.
```suggestion
    - name: "Run tests on Python 3.10"
```